snap/snapcraft.yaml

```diff
@@ -21,6 +21,7 @@ plugs:
     interface: personal-files
     write:
       - $HOME/.exoscale
+      - $HOME/.cloudstack.ini
```

Comment:
Now that I look closer, the error message relates to a file read whereas your fix is for the `write` section. Can you confirm that this change actually fixes this issue you're reporting?